src/components/PasswordStrengthMeter.vue

```diff
@@ -5,24 +5,19 @@
       class="Password__group"
     >
       <input
+        v-bind="$attrs"
         :type="inputType"
-        :ref="referanceValue"
+        :ref="referenceValue"
         :class="[defaultClass, disabled ? disabledClass : '']"
```

Comment:
👍  This is working nicely for me, besides probably needing this change:
```suggestion
        :class="[defaultClass, $attrs.disabled ? disabledClass : '']"
```